.github/workflows/build-library.yml

```diff
@@ -72,12 +72,15 @@ jobs:
           node-version: 13.3.0
           registry-url: https://registry.npmjs.org/
       - run: npm install
+      - run: npm build
       - name: Publish if version has been updated
-        uses: pascalgn/npm-publish-action@1.1.0
+        uses: oeway/npm-publish-action@97c098d
         with:
           tag_name: "v%s"
           tag_message: "v%s"
           commit_pattern: "^Release (\\S+)"
+          commit_user: oeway
```

Comment:
I think you need to add these new options as inputs in the forked action.

Reply:
What do you mean by `as inputs`?

Reply:
https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#inputs

Reply:
Got it now, just did: https://github.com/oeway/npm-publish-action/commit/2c7c8b50a9d8c19f76059fc5b6b45a27f29df3b0

Thanks.